static/js/stream_list.js

```diff
@@ -103,27 +103,36 @@ exports.build_stream_list = function (force_rerender) {
     }
 
     const parent = $("#stream_filters");
+    const pinparent = $("#pinned_filters");
     const elems = [];
+    const pinelems = [];
```

Comment:
We always use underscores between words, so we would use "pin_elems" here.  Or maybe "pinned_elems".